config-templates/redhat/odf-remote/4.8/custom-parameters.json

```diff
@@ -72,7 +72,7 @@
     "required": "false"
   },
   {
-    "description": "The IBM COS location constraint.",
+    "description": "The location constraint that you want to use when creating your bucket. For example `us-east-standard`.",
```

Comment:
Can be changed to something along the lines of --> The location of the COS instance that you want to use when creating your bucket for the noobaa backing-store of your ODF deployment